modules/core/src/test/java/org/apache/ignite/suites/CoreIgnitePdsWithIndexingCoreTestSuite.java

```diff
@@ -0,0 +1,95 @@
+import org.apache.ignite.internal.processors.cache.persistence.db.IgnitePdsPageEvictionTest;
+import org.apache.ignite.internal.processors.cache.persistence.db.IgniteSequentialNodeCrashRecoveryTest;
+import org.apache.ignite.internal.processors.cache.persistence.db.file.IgnitePdsCacheDestroyDuringCheckpointTest;
+import org.apache.ignite.internal.processors.cache.persistence.db.file.IgnitePdsCacheIntegrationTest;
+import org.apache.ignite.internal.processors.cache.persistence.db.file.IgnitePdsDiskErrorsRecoveringTest;
+import org.apache.ignite.internal.processors.cache.persistence.db.file.IgnitePdsNoActualWalHistoryTest;
+import org.apache.ignite.internal.processors.cache.persistence.db.file.IgnitePdsThreadInterruptionTest;
+import org.apache.ignite.internal.processors.cache.persistence.db.wal.IgniteWalRebalanceTest;
+import org.apache.ignite.internal.processors.cache.persistence.db.wal.IgniteWalRecoveryPPCTest;
+import org.apache.ignite.internal.processors.cache.persistence.db.wal.WalOnNodeStartTest;
+import org.apache.ignite.internal.processors.cache.persistence.db.wal.WalPathsTest;
+import org.apache.ignite.internal.processors.cache.persistence.db.wal.WalRecoveryTxLogicalRecordsTest;
+import org.apache.ignite.internal.processors.cache.persistence.db.wal.WalRolloverRecordLoggingFsyncTest;
+import org.apache.ignite.internal.processors.cache.persistence.db.wal.WalRolloverRecordLoggingLogOnlyTest;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+/**
+ * Test suite for tests that cover core PDS features and depend on indexing module.
+ */
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+    IgnitePdsCacheIntegrationTest.class,
+    IgnitePdsPageEvictionTest.class,
+    IgnitePdsMultiNodePutGetRestartTest.class,
+    IgnitePersistentStoreCacheGroupsTest.class,
+    IgnitePersistenceSequentialCheckpointTest.class,
+    PersistenceDirectoryWarningLoggingTest.class,
+    WalPathsTest.class,
+    WalRecoveryTxLogicalRecordsTest.class,
+    WalRolloverRecordLoggingFsyncTest.class,
+    WalRolloverRecordLoggingLogOnlyTest.class,
+    WalOnNodeStartTest.class,
+
+    IgnitePdsNoActualWalHistoryTest.class,
+    IgniteWalRebalanceTest.class,
+
+    IgnitePdsAtomicCacheRebalancingTest.class,
+    IgnitePdsAtomicCacheHistoricalRebalancingTest.class,
+
+    IgnitePdsTxCacheRebalancingTest.class,
+    IgnitePdsTxHistoricalRebalancingTest.class,
+
+    IgniteWalRecoveryPPCTest.class,
+
+    IgnitePdsDiskErrorsRecoveringTest.class,
+
+    IgnitePdsCacheDestroyDuringCheckpointTest.class,
+
+    IgnitePdsBinaryMetadataOnClusterRestartTest.class,
+    IgnitePdsBinaryMetadataAsyncWritingTest.class,
+    IgnitePdsMarshallerMappingRestoreOnNodeStartTest.class,
+    IgnitePdsThreadInterruptionTest.class,
+    IgnitePdsBinarySortObjectFieldsTest.class,
+
+    IgnitePdsCorruptedIndexTest.class,
+
+    IgniteLogicalRecoveryTest.class,
+
+    IgniteSequentialNodeCrashRecoveryTest.class
+})
+public class CoreIgnitePdsWithIndexingCoreTestSuite {
```

Comment:
Let's postpone this great suite renaming. It's even more confusing now than it was before.